kore/src/Kore/Attribute/Axiom.hs

```diff
@@ -97,6 +99,10 @@ data Axiom =
     , constructor :: !Constructor
     -- ^ Shows that this is one of the constructor axioms
     -- (e.g. no confusion, no junk)
+    , functional :: !Functional
```

Comment:
What does this attribute mean for axioms? Also, does this use `Kore.Attribute.Functional`? I think we only use that for symbols so the documentation should need updating if we were to use it with axioms as well.

Reply:
it is copied by the frontend to axioms stating the functionality of a symbol.

Reply:
Oh, I see. Then disregard my comment. Thanks for the explanation!

Reply:
It's actually a very good comment. I've added a note to the documentation of the functional attribute stating that it can be used for axioms, too.